### ISSUE TITLE
Text obj delete in keyboard action

### ISSUE THREAD

**Issue by Moo2017** [lvgl/lvgl#186]
Assume that I have created an screen with some buttons there once user click on one of the button a keyboard object appears , after type the desired text and press ok button , the main screen should appears , In OK action I delete the keyboard obj but still text area has remained and I don't have access to that in OK action callback function, for now I created a global variable for text area so I am able to delete it in keyboard callback function but I don't know what is the best way or there is other solutions , I hope I have had some helps here. 
I tried to get text object from the keyboard and then delete the text but it didn't work and system crashed :  lv_obj_del(lv_kb_get_ta(kb))

Is there any way to delete all existing object in the screen? 

Basically I don't know how I should move between menus, for example if I create an screen with two button and then if user click on one of the button I want to display a keyboard, and after closing the keyboard I call the first menu which cause to create the button objects again ( imagine a switch case ) , I am not sure if this cause to create many objects? 

Thanks in advance

**Comment by kisvegabor** [comment]
Once you assigned a Text area to Keyboard with `lv_kb_set_ta()` you can get that Text area in the action with `lv_kb_get_ta()`.

**Comment by Moo2017** [comment]
I did that but then system crashed and I got some error in adding object to lv_group in other screens. (Basically when in a function I delete previous objects and then create another one and add object to group I will get errors and system crashed. So after deleting the object I delete the group too , so problem been resolved)

**Comment by kisvegabor** [comment]
Please show a simple code example to reproduce the issue.

**Comment by Moo2017** [comment]
I think problem is because in lv_app_kb_action , text is de-assigned from the keyboard and that's why it's not accessible in ok_action or close_action.Also because of that if lv_kb_get_ta is called in  either ok_action or close_action callback function , system is crashed.

**Comment by kisvegabor** [comment]
You are right! I fixed it in `beta`. Is it working now?

**Comment by Moo2017** [comment]
For now I defined a global variable as an ta object and I get the text directly from the ta object. also I can edit lv_app_kb_action and remove de-assigned part if there is ok or close action.
I am good for now , thank you!

**Comment by kisvegabor** [comment]
Okay, than you for the feedback.
Then I close this issue.
